helm/values.yaml

```diff
@@ -50,6 +50,53 @@ coder:
     # coder.serviceAccount.name -- The service account name
     name: coder
 
+  # coder.podSecurityContext -- Fields related to the pod's security context
+  # (as opposed to the container). Some fields are also present in the
+  # container security context, which will take precedence over these values.
+  podSecurityContext:
+    # coder.podSecurityContext.runAsNonRoot -- Requires that containers in
+    # the pod run as an unprivileged user. If setting runAsUser to 0 (root),
+    # this will need to be set to false.
+    runAsNonRoot: true
+    # coder.podSecurityContext.runAsUser -- Sets the user id of the pod.
+    # For security reasons, we recommend using a non-root user.
+    runAsUser: 1000
+    # coder.podSecurityContext.runAsGroup -- Sets the group id of the pod.
+    # For security reasons, we recommend using a non-root group.
+    runAsGroup: 1000
+    # coder.podSecurityContext.seccompProfile -- Sets the seccomp profile
+    # for the pod. If set, the container security context setting will take
+    # precedence over this value.
+    seccompProfile:
+      type: RuntimeDefault
+
+  # coder.securityContext -- Fields related to the container's security
```

Comment:
If the pod security policy is enough for the customers that need this (and openshift) then we should just have one over the other IMO.

I would prefer the container security policy over the pod security policy if that works